### PR TITLE
Fix/UI adjustments

### PR DESCRIPTION
**Pull request by emnlpstr**
fixed a large part of ui bugs

## Checklist

- [x] I followed the indications in the [CONTRIBUTING](CONTRIBUTING.md)
- [x] The documentation related to the proposed change has been updated accordingly (also comments in code).
- [ ] Have you written new tests for your core changes, as applicable?
- [ ] Have you successfully ran tests with your changes locally?
- [x] Ready for review! :rocket:

## Fixes

- Fixes #

### CODE REVIEW

**Review by lollomag — APPROVED** [review]

File: contents/come-funziona/come-funziona.yml [hunk 1]
```diff
@@ -96,7 +94,7 @@ timeline:
       text: L’Unione Europea approva un piano da €750 miliardi che ha l'obiettivo di rilanciare l'economia europea dopo la pandemia di COVID-19 e di renderla più verde e digitale.
     - id: 2
       active: false
-      time: MAGGIO 2021
+      time: Aprile 2021
```
Comment (sebbalex) [inline contents/come-funziona/come-funziona.yml:97]: 2021?

File: src/layouts/IndexPage.jsx [hunk 2]
```diff
@@ -46,7 +47,7 @@ export const IndexPage = () => {
   useEffect(() => {
     (async () => {
       try {
-        await fetch(`https://github.com/teamdigitale/padigitale2026-opendata`)
+        await fetch(`https://raw.githubusercontent.com/tuxolino/testapi/main/data/NC002.json`)
```
Comment (sebbalex) [inline src/layouts/IndexPage.jsx:50]: please use the official url

File: contents/come-funziona/come-funziona.yml (hunk 1 above)
Comment (emnlpstr) [reply contents/come-funziona/come-funziona.yml:97]: image

in figma we have "Aprile 2021" in the card "Italia Domani"

File: src/layouts/IndexPage.jsx (hunk 2 above)
Comment (emnlpstr) [reply src/layouts/IndexPage.jsx:50]: Ok, we changed the URL

**Pull request merged**